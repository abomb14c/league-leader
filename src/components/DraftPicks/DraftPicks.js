import React from 'react'; 

const DraftPicks = ({teams}) => {
  const displayPicks = teams.map((team, index) => {
    return (
      <div key={index} className="draft-card draft-pick">
        <h3 className="draft-name">{team.team}</h3>
      </div>
    );
  });
  return (
    <div>
      {displayPicks}
    </div>
  );

};

export default DraftPicks;