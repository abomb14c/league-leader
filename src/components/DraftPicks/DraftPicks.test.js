import React from 'react';
import { shallow } from 'enzyme';
import  DraftPicks  from './DraftPicks';

describe('DraftPicks', () => {
  let wrapper;
  let mockProps;

  beforeEach(() => {
    mockProps = {
      teams: [{}, {}, {}]
    };

    wrapper = shallow(<DraftPicks
      {...mockProps}
    />);
  });
  
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});

// add test to render correct amount of teams