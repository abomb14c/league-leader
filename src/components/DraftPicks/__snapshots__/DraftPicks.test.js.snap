// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DraftPicks should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <DraftPicks
    teams={
        Array [
            Object {},
            Object {},
            Object {},
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="draft-card draft-pick"
>
          <h3
                    className="draft-name"
          />
</div>,
        <div
          className="draft-card draft-pick"
>
          <h3
                    className="draft-name"
          />
</div>,
        <div
          className="draft-card draft-pick"
>
          <h3
                    className="draft-name"
          />
</div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "0",
        "nodeType": "host",
        "props": Object {
          "children": <h3
            className="draft-name"
/>,
          "className": "draft-card draft-pick",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "draft-name",
          },
          "ref": null,
          "rendered": null,
          "type": "h3",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": "1",
        "nodeType": "host",
        "props": Object {
          "children": <h3
            className="draft-name"
/>,
          "className": "draft-card draft-pick",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "draft-name",
          },
          "ref": null,
          "rendered": null,
          "type": "h3",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": "2",
        "nodeType": "host",
        "props": Object {
          "children": <h3
            className="draft-name"
/>,
          "className": "draft-card draft-pick",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "draft-name",
          },
          "ref": null,
          "rendered": null,
          "type": "h3",
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="draft-card draft-pick"
>
            <h3
                        className="draft-name"
            />
</div>,
          <div
            className="draft-card draft-pick"
>
            <h3
                        className="draft-name"
            />
</div>,
          <div
            className="draft-card draft-pick"
>
            <h3
                        className="draft-name"
            />
</div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "0",
          "nodeType": "host",
          "props": Object {
            "children": <h3
              className="draft-name"
/>,
            "className": "draft-card draft-pick",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
              "className": "draft-name",
            },
            "ref": null,
            "rendered": null,
            "type": "h3",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": "1",
          "nodeType": "host",
          "props": Object {
            "children": <h3
              className="draft-name"
/>,
            "className": "draft-card draft-pick",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
              "className": "draft-name",
            },
            "ref": null,
            "rendered": null,
            "type": "h3",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": "2",
          "nodeType": "host",
          "props": Object {
            "children": <h3
              className="draft-name"
/>,
            "className": "draft-card draft-pick",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
              "className": "draft-name",
            },
            "ref": null,
            "rendered": null,
            "type": "h3",
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
